auth/common.cc

```diff
@@ -103,7 +103,7 @@ future<> wait_for_schema_agreement(::service::migration_manager& mm, const datab
 }
 
 const timeout_config& internal_distributed_timeout_config() noexcept {
-    static const auto t = 5s;
+    static const auto t = utils::updateable_value<db::timeout_clock::duration>(5s);
```

Comment:
Shouldn't this change go together with the previous one? Doesn't changing the values to be updateable require changes to the users to compile?

Reply:
it does

Reply:
If you mean the configuration change, then no, if a configuration is updatable you can still use it as before.